web/source/kmwosk.js

```diff
@@ -660,19 +700,40 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
         case 'K_SHIFT':
           nextLayer = 'shift'; break;
         case 'K_LCONTROL':
-        case 'K_RCONTROL':
         case 'K_LCTRL':
+          if(keymanweb.isChiral()) {
```

Comment:
I am a bit concerned about the use of K_LCONTROL and K_LCTRL and so on and so forth. I recognise this isn't new code, but why is this happening (and can we fix it)

Reply:
I'm pretty curious too, but didn't want to rock the boat there.  Some of it may have to do with layouts and/or cross-platform compatibility, I'm guessing.